packages/medusa/src/services/pricing.ts

```diff
@@ -63,32 +63,30 @@ class PricingService extends TransactionBaseService<PricingService> {
   async collectPricingContext(
     context: PriceSelectionContext
   ): Promise<PricingContext> {
-    return await this.atomicPhase_(async (manager: EntityManager) => {
```

Comment:
@srindom I have removed the atomicPhase here since it is only reading the data, it can depends on the actual manager wdyt?

Reply:
Sounds good